TypesToSerialize/BookShelf.cs

```diff
@@ -35,18 +36,20 @@ public BookShelf(string secret)
         {
             Secret = secret;
         }
-
+#if NETCOREAPP3_0_OR_GREATER
+        [MemoryPackConstructor]
```

Comment:
A polyfill would also work.

Reply:
Looks great. I will check and also update to .NET 7.0. Looks like you did again create the fastest serializer. But I will measure first ;-)

Reply:
I wish I did :) No, I'm not the author, I've just recently found out about the new project and I thought I'd add it.